src/Api/CMakeLists.txt

```diff
@@ -34,6 +34,24 @@ target_link_libraries(Api PUBLIC
         OrbitBase
         OrbitProducer
         ProducerSideChannel)
+
+
+add_library(ApiLib STATIC)
```

Comment:
I would try to find a more specific name for this target. The relationship with the `Api` is not clear from the name. Is it the target responsible for loading `Api` when the game is running? Then it could probably be called `ApiLoader`. In that case it should also be in a separate directory.

Reply:
Yeah good point, I like ApiLoader, I'll implement your suggestion.

Reply:
Will do so in another PR shortly actually, b/185530194